packages/trip-details/src/index.tsx

```diff
@@ -330,6 +338,41 @@ export function TripDetails({
             }
           />
         )}
+        {co2 > 0 && co2Config.enabled && (
+          <TripDetail
+            icon={<Leaf size={17} />}
+            summary={
+              <S.CO2Summary>
+                <FormattedMessage
+                  defaultMessage={defaultMessages["otpUi.TripDetails.co2"]}
+                  description="Text showing the quantity of CO2 emitted by this trip."
+                  id="otpUi.TripDetails.co2"
+                  values={{
+                    co2: (
+                      <FormattedNumber
+                        value={Math.round(co2)}
+                        // eslint-disable-next-line react/style-prop-object
+                        style="unit"
+                        unit={co2Config?.units || "gram"}
+                        unitDisplay="narrow"
+                      />
+                    ),
+                    strong: boldText
+                  }}
+                />
+              </S.CO2Summary>
+            }
+            description={
+              <FormattedMessage
+                defaultMessage={
+                  defaultMessages["otpUi.TripDetails.co2description"]
+                }
+                description="Text explaining how the CO2 emissions is calculated."
+                id="otpUi.TripDetails.co2description"
```

Comment:
Could we change this to include some of the values we use? For example "CO2 output is calculated by multiplying the distance of the trip (2.4km) by the average CO2 emitted by each mode involved."

Reply:
I honestly don't think that would be all that useful information, and it isn't that simple to implement. I don't have immediate access to the trip distance (but I could easily calculate it with a reducer on legs). More importantly, unless we always displayed km, this would also involve unit conversion. And I don't see it as being very useful, since at no point in the calculation are we using the total trip distance (since it's calculated per-leg).

Reply:
That makes sense! Definitely a nice-to-have so if it's a ton of effort to implement probably not worth it.

Do you think linking to our average emission info is a good idea?

Reply:
What do you mean by linking to our average emissions?

Reply:
We should probably link to the pdf we used to get the values we use to calculate the emissions the same way we do for the calories burned!

Reply:
Link added.